mobile/package.json

```diff
@@ -17,27 +17,27 @@
     "@react-navigation/native": "^6.0.10",
     "@react-navigation/native-stack": "^6.6.2",
     "dotenv": "^16.0.1",
-    "expo": "^45.0.6",
-    "expo-auth-session": "~3.6.1",
-    "expo-barcode-scanner": "~11.3.0",
-    "expo-camera": "~12.2.0",
-    "expo-constants": "~13.1.1",
-    "expo-font": "~10.1.0",
-    "expo-random": "~12.2.0",
-    "expo-splash-screen": "~0.15.1",
-    "expo-status-bar": "~1.3.0",
-    "jest-expo": "^45.0.0",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-native": "0.68.2",
+    "expo": "^46.0.15",
+    "expo-auth-session": "^3.3.1",
+    "expo-barcode-scanner": "~11.4.0",
+    "expo-camera": "~12.3.0",
+    "expo-constants": "^13.2.4",
+    "expo-font": "~10.2.0",
+    "expo-random": "~12.3.0",
+    "expo-splash-screen": "~0.16.2",
+    "expo-status-bar": "~1.4.0",
+    "jest-expo": "^42.1.0",
+    "react": "18.0.0",
+    "react-dom": "18.0.0",
+    "react-native": "0.69.6",
```

Comment:
I think updating the whole version could cause trouble for other people with older expo version. I know you're having problem with versioning and I'd suggest downgrading your expo version

Reply:
my expo version: 6.0.1